.gitignore

```diff
@@ -24,10 +24,9 @@ benchmark.txt
 npm_licenses.tar.bz2
 /web/ui/static/react
 
-/vendor
 /.build
 
-/**/node_modules
+/**/node_modules/.cache
```

Comment:
question: I see this was checked in with f3c5df45bbfb88120b2b249f61c75125e10ec4ae, why did we need to change the file at all?

Reply:
suggestion: could you add a comment explaining?

Reply:
The same way `go mod vendor` downloads and stores all files in `vendor`, NodeJS downloads and stores all files in `node_modules`. Since we want to contain all vendoring within this repository, we allow `node_modules` as we do `vendor`.  We don't need any more than that to build, so we exclude the `.cache` directory. Otherwise, this directory would grow as we continue doing NodeJS and dependency upgrades.

Does the commit text not already explain the intent?

This change was originally part of the first-ever fork commit: https://github.com/GoogleCloudPlatform/prometheus/commit/623141cf6e2b9b6aad2e1a52ad1f6cb7152c9e9c

Reply:
Ah makes sense. Thanks for clarifying!